packages/ui/sd.config.ts

```diff
@@ -32,6 +32,13 @@ StyleDictionary.extend({
       files: [
         {
           destination: 'dist/theme.css',
+          // This filter function will remove any design tokens that are marked with a deprecatedStateToken flag
```

Comment:
I do not have the entire context why we have to deprecate those state tokens without `_` and then copy them to the right buckets. I recall the generated CSS variable names should be the same because style dictionary will ignore `_`

Reply:
We are deprecating the old state tokens without `_` for consistency.The css variables created are the same between the state tokens with `_` and without, but when using the actual theme object it doesn't make much sense to have just a handful of state tokens not have `_` when all the rest of them do. The reason we are copying them into the new bucket is so that this isn't a breaking change, this allows us to introduce the new tokens without removing the old ones, the old tokens will be completely removed with the next major version release.